bin/export-record-barcodes

```diff
@@ -60,7 +60,7 @@ def main():
         ScanProject(23089, "en", "irb-kiosk"),
     ]
 
-    with ThreadPoolExecutor(8) as pool:
+    with ThreadPoolExecutor(1) as pool:
```

Comment:
if we're setting this to 1 thread, do we need `ThreadPoolExecutor()` at all?

Reply:
Nope, but I would probably leave it. We want to be making these requests in parallel, and I assume we'll be able to do so again in due course.

Reply:
That works for me.